### ISSUE TITLE
sematic preservation loss

### ISSUE THREAD

**Issue by cfeng16** [nnnth/UniLIP#22]
Hi,

Nice work! I was also trying the two-stage tokenizer training following your repo. However, during stage 2, I found that the semantic loss kept increasing, even by step 65k. I used a batch size of 256. Do you have any intuition about why this might be happening?

Thanks!

**Comment by nnnth** [comment]
I think you can try the following methods:
1. Verify that using the same `batch size` as the repository results in a normal loss.
2. Increasing the `batch size` usually requires increasing the training learning rate proportionally; otherwise, it is equivalent to lowering the effective learning rate, which may also have an impact.

**Comment by cfeng16** [comment]
Thanks for your reply! Could you let me know the global batch size you used for tokenizer stage 2 traninig?
also just to confirm, does UniLIP use LR 1e-5 for the encoder?

**Comment by nnnth** [comment]
Sorry for the late reply. As shown in [script](https://github.com/nnnth/UniLIP/blob/main/tokenizer/1b_stage2.sh), we set global batch size as 64. In the second stage, we set LR 1e-5 for the encoder.
